metastats-domain/src/main/groovy/life/qbic/metastats/exceptions/InvalidProjectStructureException.groovy

```diff
@@ -0,0 +1,26 @@
+package life.qbic.metastats.exceptions
+
+/**
+ * Invalid project structures thrown an InvalidProjectStructureException
+ *
+ * If an imported project does not follow the data model as described in doc/Openbis_Projectstructure.md this error needs to be thrown
+ *
+ * @since: 1.0
+ * @author: Jennifer Bödker
+ *
+ */
+class InvalidProjectStructureException extends IllegalArgumentException {
+
+    /**
+     * Creates an IllegalFileType exception with an error message
+     * @param error Content of error message
+     */
+    InvalidProjectStructureException(String error){
+        super(error)
+    }
```

Comment:
This constructor is not necessary since it is already a constructor of the superclass and groovy provides them of the batch.

Reply:
Imo I think its nice to have it explicitly since otherwise the class would be empty right?

Reply:
Yes. You are correct, the class would be empty but that is not a bad thing. It is more important that you construct your exceptions to include the cause (throwable).  

This is really NIT but less code -> less opportunity for errors.